src/components/Dropdown/styles/_styles.scss

```diff
@@ -0,0 +1,38 @@
+.dropdown__btn {
+	 @include cx-btn--size(34px);
+	 margin-bottom: 0px;
+	 position: relative;
+	 background-color: $gb-color-component-dark;
+	 color: black;
+	 border: none;
+	 	&:hover{
+	 		cursor: pointer;
+	 	}	  
+	}
+.dropdown__content {
+	display: block;
+	margin-top: 0px;
+	width: 200px;
+	background-color: $gb-color-component-dark;
+	color: black;
+	text-align: left;
+		ul {
```

Comment:
There is NO STYLE NESTING in BEM - please refactor, everything should be a BEM compliant.